### ISSUE TITLE
How can we load events month wise only when that month is selected

### ISSUE THREAD

**Issue by vishnuvs60** [codrops/Calendario#60]
How can we load events(data) month wise when previous or next button is clicked. 
Default I need to load only current month data. While clicking the previous or next button I need to load that months data.

**Comment by deviprsd** [comment]
Use SetData function

**Comment by vishnuvs60** [comment]
How can we clear the data? Is there any way?

**Comment by deviprsd** [comment]
Yes, https://github.com/codrops/Calendario/blob/master/js/jquery.calendario.js#L381, read the setData function. The second argument is a boolean, so if you do something like `setData(data, true)`, it will clear the old data.
